openapi/generated/api_session.yaml

```diff
@@ -182,6 +215,10 @@ definitions:
           problem.
 
           It may or may not yield further information if dereferenced.
+  UnlockCode:
+    type: string
+    minLength: 9
+    maxLength: 9
```

Comment:
```suggestion
  UnlockCode:
    type: string
    pattern: ^\d{9}$
```

Reply:
Done in [948ec83](https://github.com/pagopa/io-backend/pull/1056/commits/948ec83b530bdea78ebd362460a27d052cb2bedc)
I had to fix the pattern using a double backslash otherwise the generated type wouldn't have decoded correctly the unlock code.